scalikejdbc-core/src/main/scala/scalikejdbc/ClassNameUtil.scala

```diff
@@ -0,0 +1,19 @@
+package scalikejdbc
+
+object ClassNameUtil {
```

Comment:
Thanks, let me change this object to `private[scalikejdbc]` after merging your PR.